animation-timing/callback-exception.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+  <head>
+    <title>requestAnimationFrame callback exception reported to window.onerror</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <link rel="help" href="https://w3c.github.io/web-performance/specs/RequestAnimationFrame/Overview.html#dom-windowanimationtiming-requestanimationframe"/>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+      var custom_exception = 'requestAnimationFrameException';
+      async_test(function (t) {
+        setup({allow_uncaught_exception : true});
+        window.onerror = function(e) {
+          t.step(function() {
+             assert_equals(e,'Error: '+custom_exception);
```

Comment:
The message property isn't well-defined; see step 3 of <https://html.spec.whatwg.org/multipage/#runtime-script-errors>. I'm not quite sure how widely implemented the fifth argument is; another alternative is to use `addEventListener("error", ...)`.